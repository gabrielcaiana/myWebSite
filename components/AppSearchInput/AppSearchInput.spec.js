import { mount } from '@vue/test-utils'
import AppSearchInput from '@/components/AppSearchInput.vue'

describe('AppSearchInput', () => {
  const wrapper = mount(AppSearchInput)

  it('is a Vue instance', () => {
    expect(wrapper.vm).toBeTruthy()
  })

  it("create Snapshot", () => {
    expect(wrapper.element).toMatchSnapshot()
  })
})
