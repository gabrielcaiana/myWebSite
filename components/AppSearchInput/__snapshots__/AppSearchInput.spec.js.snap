// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppSearchInput create Snapshot 1`] = `
<div
  class="z-50"
>
  <input
    autocomplete="off"
    placeholder="Search Articles"
    type="search"
  />
   
  <!---->
</div>
`;
